packages/list-view/tests/controller_test.js

```diff
@@ -43,31 +43,40 @@ module("Ember.ListView controllers", {
 
 test("parent controller", function() {
   var watWasCalled = false;
-  var controller = Ember.Controller.extend({
-    foo: 'bar',
-    actions: {
-      wat: function() {
-        watWasCalled = true;
+  var controller;
+
+  Ember.run(function(){
```

Comment:
My guess is that the `create` caused the requirement to wrap in a run-loop, not the `extend`.